packages/core/src/icon/icon.element.scss

```diff
@@ -36,105 +36,6 @@ svg {
   display: block;
 }
 
-// sizing
-// DEPRECATED IN 4.0: all legacy sizing classnames
-// will be replaced with [size="sm|md|lg|xl|xxl"] selectors!
-:host(.clr-i-size-sm) {
```

Comment:
The `.clir-i-size-*` classes have been removed but have not been replaced. This will break sizing across all icons.

Please fix.

Also, there are a lot of deprecations in this file that were missed (e.g. `has-badge`, `has-alert`, etc.)

Reply:
What do you mean by `not been replaced`? 

Do you mean `:host([size="sm"])` instead of `:host(.clr-i-size-sm)`,  `:host([size="md"])` instead of `:host(.clr-i-size-md)`, etc ?

Reply:
Yes. Also the code that "translates" `size="sm"` into the classnames is no longer needed.